src/pages/team/index.tsx

```diff
@@ -104,7 +103,7 @@ const AtCapacityNotice = ({teamData}: {teamData: TeamData}) => {
               onClick={() => {
                 track(`clicked manage membership`)
               }}
-              className="transition-all duration-150 ease-in-out text-white font-semibold rounded-md"
+              className="transition-all duration-150 ease-in-out font-semibold rounded-md dark:text-yellow-400 dark:hover:text-yellow-300"
```

Comment:
I'm struggling to get this bit of dark-mode styling to apply. I don't know how tailwindcss precedence is determined, but the prose dark-mode light blue link style is taking precedence over these element-specific styles.

The yellow looks a lot better against the burnt orange, than the light blue does.

@DrShpongle @vojtaholik any ideas on how to make the yellow apply?